HungryforApples/app/src/main/java/com/exeinformatique/hungryforapples/MainActivity.java

```diff
@@ -80,5 +88,22 @@ private void generateMarkers() {
             }
         }
         return position;
+
+        setListeners();
```

Comment:
**Cause une erreur: _Out of reach_** (Il faut qu'il soit dans le _onCreate_, pas la fonction _getRestaurantsPosition_